support-figma/extended-layout-plugin/manifest.json

```diff
@@ -21,13 +33,20 @@
         { "name": "Export Images", "command": "export-images" },
         { "name": "Clear Image Resource", "command": "clear-image-res" }
       ]
+    },
+    { "separator": true },
+    {
+      "name": "Shader",
+      "command": "shader"
     }
+
   ],
   "networkAccess": {
     "allowedDomains": [
       "https://cdn.jsdelivr.net/gh/thomas-lowry/figma-plugin-ds/dist/figma-plugin-ds.css",
       "https://rsms.me",
-      "https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"
+      "https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js",
+      "https://unpkg.com"
     ],
     "reasoning": "The plugin uses the stylesheet from https://github.com/thomas-lowry/figma-plugin-ds to make the UI consistent with the rest of the Figma UI (https://www.figma.com/plugin-docs/figma-components/). It uses https://rsms.me for the Inter fonts. jszip is the tool to download images in zip."
```

Comment:
Please add a bit explaining the need for unpkg.com

Reply:
Done.